src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -322,8 +318,8 @@ private MinigameTask cloneBugfinder(final MinigameTask minigameTask, final Strin
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("finitequiz-backend not present")) {
-                    e.printStackTrace();
+                if (!errorMessages.contains("bugfinder-backend not present")) {
+                    log.debug(e.getMessage());
```

Comment:
Would `log.debug(e)` be possible as well?
Alternatively, you can of course use `log.debug("Encountered Exception", e)`